### PR TITLE
cleanup subspace

### PR DESCRIPTION
**Pull request by bluejekyll**
There were a couple of things from the subspace commit that I wanted to cleanup, related to unwraps and naming. Let me know what you think.

### CODE REVIEW

File: foundationdb/src/subspace.rs [hunk 1]
```diff
@@ -68,16 +68,16 @@ impl Subspace {
     /// removed.  `unpack` will return an error if the key is not in this Subspace or does not
     /// encode a well-formed Tuple.
     pub fn unpack<T: Tuple>(&self, key: &[u8]) -> Result<T, TupleError> {
-        if !self.contains(key) {
```
Comment (yjh0502) [inline foundationdb/src/subspace.rs]: I think name `contains` is more consistent with other APIs.

https://apple.github.io/foundationdb/api-python.html#subspaces
https://apple.github.io/foundationdb/javadoc/com/apple/foundationdb/subspace/Subspace.html#contains-byte:A-

Comment (bluejekyll) [reply foundationdb/src/subspace.rs]: I was aware. I guess we have a choice, be consistent with the others, or be more accurate. I feel like contains is very misleading and wrong. I’m open to other names, like ‘is_prefix_of’

Comment (yjh0502) [reply foundationdb/src/subspace.rs]: I like `covers` than `is_prefix_of` (BTW I 
prefers `starts_with` in this case, as it is what we are using in the impl). Thanks for the comment! I'll merge after CI.

Comment (rushmorem) [reply foundationdb/src/subspace.rs]: I also think that `starts_with` would be the perfect name for this. It's also consistent with the `std` library, making it more intuitive.

**Pull request merged**